### PR TITLE
Remove any compatibility hack with the 1.x line from 2.0

### PR DESCRIPTION
**Pull request by emanuele45**

### CODE REVIEW

File: install/install.php [hunk 1]
```diff
@@ -81,9 +81,6 @@ function initialize_inputs()
 	if (ini_get('magic_quotes_sybase') || strtolower(ini_get('magic_quotes_sybase')) == 'on')
```
Comment (live627) [inline install/install.php]: kill this too

File: install/installcore.php [hunk 2]
```diff
@@ -13,7 +13,7 @@
 const CURRENT_VERSION = '1.1';
 const CURRENT_LANG_VERSION = '1.1';
 const DB_SCRIPT_VERSION = '1-1';
-const REQUIRED_PHP_VERSION = '5.3.3';
+const REQUIRED_PHP_VERSION = '5.5.0';
```
Comment (Frenzie) [inline install/installcore.php]: If this is the case you should also update the README.

Comment (emanuele45) [reply install/installcore.php]: Good catch, done!

File: install/install.php (hunk 1 above)
Comment (emanuele45) [reply install/install.php]: I think I left it as a warning, but... yeah, doesn't make much sense I guess.
Removed.

**Comment by emanuele45** [comment]
@live627 I saw the SessionHandler class, if I understood correctly, with PHP 7.x the whole class can just be removed and used the PHP built in, right?

**Comment by live627** [comment]
PHP 5.4, but yes, drop in replacement.

**Comment by emanuele45** [comment]
Good, then I'll remove it as well before merging the branch I guess.

**Comment by emanuele45** [comment]
I removed everything I could find along with all the hacks for any php < 7... I think. :laughing:

**Pull request merged**